deploy/docs/v1_migration_doc.md

```diff
@@ -0,0 +1,178 @@
+# Kubernetes Collection 1.0.0 - Breaking Changes
```

Comment:
We should add the rollback information:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/601#discussion_r421097694